### ISSUE TITLE
Parsing error at boot with expindustry:casting_mold

### ISSUE THREAD

**Issue by LemADEC** [Draco18s/ReasonableRealism#24]
As of 1.12.1-Harderfarming-4.0.0d, there's an exception during boot:
```
[17:58:20] [Server thread/ERROR] [FML]: Parsing error loading recipe harderfarming:mold_knife
com.google.gson.JsonSyntaxException: Unknown item 'expindustry:casting_mold'
	at net.minecraftforge.common.crafting.CraftingHelper.getItemStack(CraftingHelper.java:212) ~[CraftingHelper.class:?]
	at com.draco18s.hardlib.api.recipes.RecipeToolMold$Factory.parse(RecipeToolMold.java:254) ~[RecipeToolMold$Factory.class:?]
	at net.minecraftforge.common.crafting.CraftingHelper.getRecipe(CraftingHelper.java:409) ~[CraftingHelper.class:?]
	at net.minecraftforge.common.crafting.CraftingHelper.lambda$loadRecipes$22(CraftingHelper.java:711) ~[CraftingHelper.class:?]
	at net.minecraftforge.common.crafting.CraftingHelper.findFiles(CraftingHelper.java:822) ~[CraftingHelper.class:?]
	at net.minecraftforge.common.crafting.CraftingHelper.loadRecipes(CraftingHelper.java:668) ~[CraftingHelper.class:?]
	at java.util.ArrayList.forEach(ArrayList.java:1257) [?:1.8.0_171]
	at net.minecraftforge.common.crafting.CraftingHelper.loadRecipes(CraftingHelper.java:620) [CraftingHelper.class:?]
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:742) [Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.finishServerLoading(FMLServerHandler.java:108) [FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStarted(FMLCommonHandler.java:337) [FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:219) [nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
```

**Comment by Draco18s** [comment]
Thanks for the report. Are you running only Harder Farming and not Expanded Industry?

**Comment by LemADEC** [comment]
yes, only Harder Farming, no Expanded Industry

**Comment by Draco18s** [comment]
\*nod\*
I think I know what the issue is, but it'll be troublesome to resolve, due to how recipes work (conditional recipes are a pain).

If you delete the recipe out of the jar file (just open it in a zip program and navigate to assets/recipes/ and delete mold_knife.json) that'll work for now.

**Comment by Draco18s** [comment]
After fixing an unrelated issue (a bit of code that had never been meant to keep, got saved) this turned out to be easier than I thought; recipe factory can just invoke a `BlankRecipe` when Exp Ind isn't installed. Will require an update to HardLib (and thus, everything else). Closing this as resolved, though I'm not sure how quickly I'll get out a new release, I've got another open issue to investigate as well and it deals with inter-mod compatibility.
